tests/library/trace-viewer.spec.ts

```diff
@@ -289,6 +289,14 @@ test('should show snapshot URL', async ({ page, runAndTrace, server }) => {
 });
 
 test('should popup snapshot', async ({ page, runAndTrace, server }) => {
+  const traceViewer = await runAndTrace(async () => {
+    await page.goto(server.EMPTY_PAGE);
+  });
+  await traceViewer.snapshotFrame('page.goto');
+  await expect(traceViewer.page.getByTestId('snapshot-breadcrumb')).toHaveText('page.goto');
```

Comment:
Ideally I'd test that `test.step` nesting works, but it seems like we don't have coverage for that / if I use `test.step` it does not end up as nesting inside the runAndTrace trace-viewer.